### PR TITLE
Typo fix phenomenon/phenomena

### PR DESCRIPTION
**Pull request by nothing-but-the-rain**
## Summary
{{Phenomenon/phenomena type fix.}}

### CODE REVIEW

**Comment by mOctave** [comment]
I'm not sure about the first change, but the second one is definitely a good one to have.
Except...
There's a bit of an ongoing/concluding conversation in https://github.com/endless-sky/endless-sky/pull/8689 about Remnant intentional grammar weirdness, which is closely related. It's probably still best to have at least the second change, but @Zitchas would know best.

**Comment by nothing-but-the-rain** [comment]
> "Black holes are such a wonderful phenomenon."
I could live with this. All black holes are the same phenomenon.

> "People more knowledgeable than me in the subject are examining the phenomena right now."
This however is not just quirky, this is incorrect. They've studied the black holes (phenomena) for ages, they're examining _this_ specific black hole (phenomenon) _right now_. 

Assuming that I'm correct with the second one, that would make the first one sound much weirder. Because they're examining this phenomenon right now (because it is apparently different), yet when they are talking about black holes, they are all the same "wonderful phenomenon".

So that's why it was bothering me, I'm not hell bent on changing this, I just thought it should be pointed out.

**Comment by EjoThims** [comment]
Well, as written, the second usage is the phenomenon of the system appearing, not of the particular black hole or it's distinctions.

Best form may be to use a different word altogether or even clarify the phrasing in general, but, as PRed, the second change is both correct and necessary.

First could go either way, so should probably be left as originally is unless it's not conveying the meaning intended by @beccabunny .

File: data/kahet/kahet missions.txt [hunk 1]
```diff
@@ -372,15 +372,15 @@ mission "Ka'het: Patir Mystery 1"
 					goto wait
 			`	"I am afraid there is not. Give me, say, half an hour and I will have the results ready."`
 			label wait
-			`	Almost an hour passes, but Taely finally responds. "The scan is complete. The system is currently processing all the data it gathered into something usable; there's nothing we can do but wait." She starts staring out of the cockpit windows. "Black holes are such a wonderful phenomenon. They are so mysterious, and each one is peculiar in its own way, this one especially."`
+			`	Almost an hour passes, but Taely finally responds. "The scan is complete. The system is currently processing all the data it gathered into something usable; there's nothing we can do but wait." She starts staring out of the cockpit windows. "Black holes are such wonderful phenomena. They are so mysterious, and each one is peculiar in its own way, this one especially."`
```
Comment (W1zrad) [inline data/kahet/kahet missions.txt:375]:
```suggestion
			`	Almost an hour passes, but Taely finally responds. "The scan is complete. The system is currently processing all the data it gathered into something usable; there's nothing we can do but wait." She starts staring out of the cockpit windows. "Black holes are such a wonderful phenomenon. They are so mysterious, and each one is peculiar in its own way, this one especially."`
```

**Review by Zitchas — APPROVED** [review]
From a Remnant perspective, I would be inclined to say that Black holes should be a phenomena, since they are plural, and the Remnant definitely don't treat all black holes as just copies of each other (each black hole is a unique thing).

That being said, for the actual unique change that is in this PR right now, changing the phenomena to phenomenon would be correct, as this particular thing appearing is a single phenomenon.

**Review by W1zrad — APPROVED** [review]

**Pull request merged**